docs/en/administrator-guide/http-actions/compaction-action.md

```diff
@@ -95,4 +95,96 @@ curl -X GET http://192.168.10.24:8040/api/compaction/show?tablet_id=10015\&schem
 
 ## Manually trigger Compaction
 
-(TODO)
+```
+curl -X GET http://be_host:webserver_port/api/compaction/run?tablet_id=xxxx\&schema_hash=yyyy\&compact_type=cumulative
```

Comment:
Why use GET method? I think POST method is more suitable.

Reply:
The document is wrong and has been modified